cromshell

```diff
@@ -0,0 +1,311 @@
+	return $( checkPipeStatus )
+}
+
+# Get the logs of a Cromwell job UUID
+function logs() {
+  curl -s ${CROMWELL_URL}/api/workflows/v1/${1}/logs | jq . ;
+	return $( checkPipeStatus )
+}
+
+# Get the metadata for a Cromwell job UUID
+function metadata() {
+  curl --compressed -s ${CROMWELL_URL}/api/workflows/v1/${1}/metadata?${CROMWELL_METADATA_PARAMETERS} | jq . ;
+	return $( checkPipeStatus )
+}
+
+# Get the metadata for a Cromwell job UUID
+function execution-status-count() {
+  curl --compressed -s ${CROMWELL_URL}/api/workflows/v1/$1/metadata?${CROMWELL_METADATA_PARAMETERS} | \
+  jq '.calls | to_entries | map({(.key): .value | flatten | group_by(.executionStatus) | map({(.[0].executionStatus): . | length}) | add})'
+	return $( checkPipeStatus )
+}
+
+function slim-metadata() {
+  curl --compressed -s "${CROMWELL_URL}/api/workflows/v1/$1/metadata?includeKey=executionStatus&includeKey=backendStatus" | jq .;
+	return $( checkPipeStatus )
+}
+
+function timing() {
+  open ${CROMWELL_URL}/api/workflows/v1/${1}/timing;
+	return $?
+}
+
+function abort() { 
+  response=$(curl -X POST --header "Content-Type: application/json" --header "Accept: application/json" "${CROMWELL_URL}/api/workflows/v1/${1}/abort")
+	local r=$?
+  echo $response  
+	return $r
+}
+
+# Spin off a task in the background that will check periodically if the workflow ID is 
+# complete.
+# When it is, send an email to the email address specified.
+function notify() {
+
+	local id=$1
+	local email=$2
+
+	error "Spinning off notification thread for workflow ${id} to ${email} ..."
+	
+	statusFile=$( mktemp )
+
+	local separator="=================================================="
+
+	while true ; do 
+		status $id 1>$statusFile
+		grep -oE "Succeeded|Failed|Aborted" $statusFile &>/dev/null
+		r=$?
+		if [ $r -eq 0 ] ; then
+			metaData=$( metadata $id )
+			echo -e "CROMWELL Task Complete:\n\n${id}\n\non\n\n${CROMWELL_URL}\n\n${separator}\n\nStatus:\n$(cat ${statusFile})\n\n${separator}\nMetadata:\n${metaData}\n\n${separator}\nSent by $( whoami )@$( hostname ) on $( date ) \n\n\n" | mail -n -s "Cromwell Task Complete [${CROMWELL_URL}]" ${email} 
+			break
+		fi
+		# wait for 10 seconds:
+		sleep 10
+	done 
+}
+
+function runSubCommandOnWorkflowId() {
+	# Get the workflow ID:
+	populateWorkflowId $1
+	error "using workflow-id == $WORKFLOW_ID"
+	${SUB_COMMAND} ${WORKFLOW_ID} 
+	shift
+	echo
+}
+
+################################################################################
+
+#Read args:
+while [ $# -gt 0 ] ; do
+
+  case "$1" in
+    -h|--h|--help|-help|help)
+      usage;
+      exit 0;
+      ;;
+    submit|status|logs|execution-status-count|metadata|slim-metadata|timing|abort|notify) 
+			# Get our sub-command:
+			SUB_COMMAND=$1
+			shift
+			# We'll manually handle the rest of the options:
+			break
+			;;
+    *)
+      error "${SCRIPTNAME}: invalid option: $1" 
+      simpleUsage
+      echo "Try \`$SCRIPTNAME --help' for more information."
```

Comment:
What's this escaped backtick for?

Reply:
To display as a back tick.  It's showing the command to run to see the options for the script in quotes, so it'll be rendered as ``cromshell --help'.